ontopy/graph.py

```diff
@@ -720,10 +720,12 @@ def _relation_styles(
         """
         for relation in entity.mro():
             if relation in rels:
+                print(relation)
+                print(get_label(relation))
                 if get_label(relation) in relations:
                     rattrs = relations[get_label(relation)]
                 else:
-                    for alt_label in relation.get_annotations()["altLabel"]:
+                    for alt_label in relation.get_annotations()["altLabel"].en:
                         rattrs = relations[alt_label]
```

Comment:
Hmm, wouldn't it be better to make item access work with locstr objects? In that case we wouldn't need to change anything here.

I think it would be relatively easy, just make get_by_label() accept locstr as input (adding a `label = str(label)`, maybe together with an optional argument for only considering a given language).